playwright-e2e/tests/pancan/enrollment/adult-self-enrollment.spec.ts

```diff
@@ -64,7 +64,7 @@ test.describe('Enroll myself as adult', () => {
     await consentFormPage.signature().fill(`${user.patient.firstName} ${lastName}`);
     await expect(consentFormPage.getSubmitButton()).toBeEnabled();
 
-    await consentFormPage.fillInContactAddress({ fullName: `${user.patient.firstName} ${lastName}`, phoneLabel: 'Phone' });
+    await consentFormPage.fillInContactAddress({ fullName: `${user.patient.firstName} ${lastName}`, labels: { phone: 'Telephone Contact Number',country: 'Country',state:'City',zip:'Zip Code', city:'City'}});
```

Comment:
Because `fillInContactAddress` has a signature change to accommodate different labels beyond just the phone field, there were some updates to existing tests to get them back to a passing state.